ERCS/erc-7592.md

```diff
@@ -0,0 +1,43 @@
+---
+eip: 7592
```

Comment:
```suggestion
eip: 7618
```
Assigning next sequential EIP/ERC/RIP number.  7592 was already assigned to another ERC.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.